crml/staking/src/lib.rs

```diff
@@ -1370,6 +1373,32 @@ impl<T: Trait> Module<T> {
 		imbalance
 	}
 
+	/// Payout transaction rewards to all the validators. Called at the beginning of an era
+	fn transaction_reward_validators() {
+		let validators = Self::current_elected();
+		let total_reward = Self::current_era_transaction_rewards();
```

Comment:
If we want to avoid making the getter for this public (which I think is what Jordy is saying above) we can use `< CurrentEraTransactionRewards <T>>::get()` instead of `Self::current_era_transaction_rewards()`

Reply:
Removed the getting function.